Need help

metelska-anastasiia opened the pull request.

metelska-anastasiia committed:
Need help

Ivan95kos reviewed (changes requested):
left advice in the chat

metelska-anastasiia committed:
To the project were added:
- all needed dtos
- repository, service, client, controller and mapper
- all required dependencies in pom.xml
- changed application.properties

kozhukhovsky commented on src/main/java/mate/academy/rickandmorty/mapper/CharacterMapper.java:
```diff
@@ -0,0 +1,14 @@
+
+@Mapper(componentModel = "spring")
+public interface CharacterMapper {
+
```
```suggestion
```

kozhukhovsky commented on src/main/java/mate/academy/rickandmorty/service/CharacterService.java:
```diff
@@ -0,0 +1,12 @@
+import org.springframework.data.domain.Pageable;
+
+public interface CharacterService {
+
```
```suggestion
```
